### PR TITLE
Use standard lib methods

### LINKED ISSUE #405: [Templates]: Use standard-libary methods
We have implemented some custom methods that are already in the core language, to make the templates easier to maintain, there is a need to start using the standard methods.

This involves going through all of the practice exercise templates and replacing `to_capitalized` with the built-in method `camelcase`.
And `to_snake` with the built in method `underscore`.

### PR DESCRIPTION
**Pull request by ls1955**
This commit:
* Refactor GeneratorHelp#to_capitalized
* Remove GeneratorHelp#to_snake
* Replace GeneratorHelp#to_snake with String#underscore across test_template.ecr

Associated Issue:
Closes #405

### CODE REVIEW

**Comment by ls1955** [comment]
Hmm... Can't figure out the error within space-age exercise.

**Comment by ls1955** [comment]
After taking a look around, It might be because we are dealing with int and float in space-age exercise?

```Crystal
num_one = 1000000000.to_s
num_two = 31.69.to_s

to_snake(num_one) # => "_1_0_0_0_0_0_0_0_0_0"
num_one.underscore # => "1000000000"
to_snake(num_two) # => "_3_1_._6_9"
num_two.underscore # => "31.69"
```

File: bin/generator_help.cr
Comment (ryanplusplus) [inline bin/generator_help.cr:1]: We can remove `to_snake` now, right?

**Review by ryanplusplus — CHANGES_REQUESTED** [review]

File: bin/generator_help.cr
Comment (ryanplusplus) [reply bin/generator_help.cr:1]: Never mind, I see it's still used in `space-age`. Let me see if I can find a way to use `String#underscore` in `space-age`.

**Review by ryanplusplus — APPROVED** [review]

**Pull request merged**